### PR TITLE
CACTUS-872: Test Cleanup

### PR DESCRIPTION
**Pull request by daniloPenaR**
[CACTUS-872](https://repayonline.atlassian.net/browse/CACTUS-872)

Nothing too much to add. 
There's a _special_ case at `Box` component.

### Testing
Run `yarn web test`, every test of every component should pass.

### CODE REVIEW

File: modules/cactus-web/src/Box/Box.test.tsx [hunk 1]
```diff
@@ -42,12 +41,11 @@ describe('component: Box', (): void => {
     expect(style.borderRadius).toBe('20px')
   })
 
+  // THos should Be solved first !!!!
```
Comment (Dhalton) [inline modules/cactus-web/src/Box/Box.test.tsx:44]: What?

**Review by wilysword — COMMENTED** [review]
This must have been pretty boring; I applaud you for getting through it.

File: modules/cactus-web/src/Select/Select.test.tsx [hunk 2]
```diff
@@ -22,18 +21,16 @@ describe('component: Select', () => {
...
     )
 
     expect(getByText('Who?')).not.toBeNull()
   })
 
   test('can receive options as children, with altText', () => {
-    const { queryByText } = render(
-      <StyleProvider>
+    const { queryByText } = renderWithTheme(
+      <>
```
Comment (Dhalton) [inline modules/cactus-web/src/Select/Select.test.tsx:33]: I don't think this fragment is needed

**Review by Dhalton — APPROVED** [review]
I'm not sure what that comment means and I think there's a `Fragment` in there that we don't need, but overall it looks good.

It looks like this probably takes care of all of the tests, is that correct?

**Comment by Dhalton** [comment]
> Also, here in that test, there's a `rerender` function that is redefining the theme object, a special case that is not present in the other tests

Yeah, the `renderWithTheme` function kind of assumes that you'd rerender with the same theme as the initial render. If you want, I bet it'd be possible to wrap the `rerender` function in another function that accepts an optional theme config argument. Then if that extra argument exists, we could generate another theme, otherwise use the first one. But this seems to be a one-off case, and I'm not super fussed about it, so I'm fine with leaving it. My approval still stands.

**Review by wilysword — APPROVED** [review]

**Pull request merged**